index.d.ts

```diff
@@ -41,9 +41,8 @@ declare module 'react-helmet-async' {
     titleTemplate?: string;
   }
 
-  export class Helmet extends React.Component<HelmetProps> {
-  }
-  
+  export class Helmet extends React.Component<HelmetProps> {}
```

Comment:
I think this one also needs `PropsWithChildren` type

Reply:
I am super sure I added it there 😆

Reply:
It could be you didn't stage it :stuck_out_tongue: